dynoid/dynoidtest/dynoidtest.go

```diff
@@ -21,18 +21,17 @@ import (
 )
 
 const (
-	// IssuerHost is the host used by the dynoidtest.Issuer
-	IssuerHost = "heroku.local"
-
-	DefaultSpaceID = "test"                                 // space id used when one is not provided
-	DefaultAppID   = "00000000-0000-0000-0000-000000000001" // app id used when one is not provided
-	DefaultAppName = "sushi"                                // app name used when one is not provided
-	DefaultDyno    = "web.1"                                // dyno used when one is not provided
+	DefaultIssuerHost = "heroku.local"                         // issuer host used when one is not provided
```

Comment:
This was misleading before. The issuer host is "oidc.heroku.local" so this should probably be the Heroku host:

```suggestion
	DefaultHerokuHost = "heroku.local"                         // issuer host used when one is not provided
```